src/web/kestraFsProvider.ts

```diff
@@ -27,29 +27,36 @@ const fileStatFromKestraFileAttrs = ({ fileName, type, creationTime, lastModifie
 
 const EXCLUDED_FOLDERS = [".git", ".vscode"];
 const DEFAULT_DIRECTORY_SIZE = 4096;
+const AUTHENTICATION_EXPIRED_ERROR = "Permission issue while calling Kestra's API. Please reload the page to reauthenticate (your changes are kept locally by VSCode cache).";
```

Comment:
What happens if authentication expired on local vscode ? (for example, schema out of date, token expired)

Reply:
Currently you don't use Kestra's schema (and so Kestra's file system provider) from desktop VSCode. It doesn't require much but right now only experimented users would be able to access Kestra FS by providing a workspace URI directly (which would start with 'kestra:') to access such file system. But even then we would have to provide a way to configure the JWT token to use since right now these calls rely on the JWT token from the web app (since cookies are brought to the VSCode web worker).

I think it's a second step and we would have to think about what's the best way to access Kestra FS from desktop. Maybe it would be a specific command to add a JWT token but then the JWT should be stored locally in VSCode :)